data/mods/realguns/gunmod.json

```diff
@@ -37,7 +37,7 @@
     "color": "green",
     "location": "bore",
     "mod_targets": [ "rifle", "shotgun" ],
-    "acceptable_ammo": [ "762", "223", "308", "shot" ],
+    "acceptable_ammo": [ "shot" ],
```

Comment:
Now that the crash issue is gone, I suppose that means I can implement the old conversion options without requiring I add dozens of weapon overrides, so I can undo this change if it is desirable.